### PR TITLE
Adjust CLI to support host/port (fix #84)

### LINKED ISSUE #84: Uunexpected keyword argument 'address'
Hi

I've just download this package and when I try to use it I got this following error message `unexpected keyword argument 'address'`

My command is `yarn_client --host <host_ip> --port 8088 nm info`

Don't understand why the code does not expect address at line 252 as address is defined just above.

### PR DESCRIPTION
**Pull request by dimon222**
To fix #84

### CODE REVIEW

**Comment by dimon222** [comment]
I think failing python2.7 tests can be ignored as its EOL and one of transitive dependencies no longer compatible with python2.7

**Comment by kevin-bates** [comment]
Regarding the PR, this seems a little funky to me.  The endpoint I need to hit is `http://yarn-eg-node-1.fyre.ibm.com:8088/ws/v1/cluster` (I suspect the `ws/v1/cluster` portion may be due to an older Hadoop, I'm running 2.7 from 2 years ago)

To get this to work via `yarn_client` after the endpoints changes, I must invoke:
```
yarn_client --host http://yarn-eg-node-1.fyre.ibm.com --port 8088/ws/v1 rm info
```
There are two things odd about this...
1. I have to decompose the endpoint into, essentially, invalid host and port pieces.
2. We automatically append `/cluster` to the provided endpoint.

It seems like updating `yarn_client` to take a `--endpoint` option would be an improvement.  And, regarding 2, if we're asking for the endpoints, why are we appending anything at all?  (I can't recall if this was discussed previously, but I'm afraid I fell privy to the default behavior to just look things up in the local config, so haven't been passing any endpoint. - Which also makes me wonder if `yarn_client` _could_ do the same thing if "endpoint information" isn't provided.)

Regarding the python 2.7 tests.  I'm inclined to drop 2.7 entirely (not sure we have a choice anyway).  Yes, I'm hitting a Hadoop cluster running on 2.7, but I'm hitting it from a python 3.x client.

**Comment by dimon222** [comment]
@kevin-bates I was mainly concerned about keeping backward compatibility, but considering CLI been already broken for quite some time, it's probably fine to break compatibility.

**Comment by kevin-bates** [comment]
@dimon222 - I appreciate that approach.  
@lucienfregosi - What's your affinity to the current CLI tooling?  Are you okay with adjusting parameters to match the python API (more or less)?

**Comment by dimon222** [comment]
@kevin-bates i've rushed and applied the change with new parameter with example:
image

**Comment by lresende** [comment]
I'm ok with both `host` and `port` or `endpoint` or both.

File: yarn_api_client/main.py [hunk 1]
```diff
@@ -15,8 +15,7 @@ def get_parser():
     parser = argparse.ArgumentParser(
         description='Client for Hadoop® YARN API')
 
-    parser.add_argument('--host', help='API host')
-    parser.add_argument('--port', help='API port')
+    parser.add_argument('--endpoint', help='API endpoint (https://test.cluster.com:1234)')
```
Comment (lresende) [inline yarn_api_client/main.py:18]: Could we use the default port in the help text?

Comment (dimon222) [reply yarn_api_client/main.py]: Changed to default RM https port.

**Pull request merged**